cli/tools/installer.rs

```diff
@@ -995,10 +1098,14 @@ mod tests {
     }
 
     // create extra files
-    file_path = file_path.with_extension("tsconfig.json");
-    File::create(&file_path).unwrap();
-    file_path = file_path.with_extension("lock.json");
```

Comment:
Bug was also happening here where it would do `echo_test.tsconfig.json` then `echo_test.tsconfig.lock.json`, which is why this test didn't catch the bug.